src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -3310,6 +3343,173 @@ .maxstack  3
 ");
         }
 
+        [Fact]
+        public void ArraysSupport()
+        {
+            var verifier = CompileAndVerifyFunctionPointers(@"
+using System;
+unsafe class C
+{
+    public static void M(string s) => Console.Write(s);
+    public static void Main()
+    {
+        delegate*<string, void>[] ptrs = new delegate*<string, void>[] { &M, &M };
```

Comment:
>{ &M, &M } [](start = 71, length = 10)

Consider testing initialization with null values and other constants we allow. Are we going to initialize the array with a blob? I think we are doing this for int starting with 3 elements.